puzzle/utils/constants.py

```diff
@@ -49,6 +49,63 @@
   'intergenic_variant'
 )
 
+IMPACT_SEVERITIES = {
```

Comment:
This is the same as `SO_TERMS `, no? If so we could avoid repeating them and just get them as the `IMPACT_SEVERITIES.keys()` given that we make it into an `OrderedDict`